tests/contest/runtimetest/src/tests.rs

```diff
@@ -545,3 +545,34 @@ pub fn test_io_priority_class(spec: &Spec, io_priority_class: IOPriorityClass) {
         eprintln!("error ioprio_get expected priority {expected_priority:?}, got {priority}")
     }
 }
+
+pub fn validate_masked_paths(spec: &Spec) {
+    let linux = spec.linux().as_ref().unwrap();
+    let masked_paths = match linux.masked_paths() {
+        Some(p) => p,
+        None => {
+            eprintln!("in readonly paths, expected some readonly paths to be set, found none");
+            return;
+        }
+    };
+
+    if masked_paths.is_empty() {
+        return;
+    }
```

Comment:
Should we error here also like above?

Reply:
In the [Go implementation](https://github.com/opencontainers/runtime-tools/blob/master/cmd/runtimetest/main.go#L904-L907), it is skipped, so it doesn't seem necessary to return an error here. However, since entering this process would be problematic for the test cases we are considering, I will add an error.
[62e5e1a](https://github.com/containers/youki/pull/2950/commits/62e5e1a1a7bd04ea1a06275661f0325ea75da37f)